test/e2e/shared_vars.go

```diff
@@ -0,0 +1,11 @@
+//go:build scale || e2e
```

Comment:
I have actually put the location info in framework/common/common.go, and also made the changes to scale file, we can do the same for createInfra and deleteInfra as well

Reply:
Done